components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -5,17 +5,22 @@
 package mozilla.components.concept.engine
 
 import android.support.annotation.CallSuper
+import android.view.View
 import mozilla.components.support.base.observer.Observable
 import mozilla.components.support.base.observer.ObserverRegistry
 
 /**
  * Class representing a single engine session.
  *
  * In browsers usually a session corresponds to a tab.
+ *
+ * @param fullScreenCallback Holds onto a callback to be invoked when fullscreen mode is exited.
```

Comment:
This should be `@property` and docs probably just `A callback to be invoked when...`